Pipfile

```diff
@@ -21,6 +21,7 @@ fuzzywuzzy = {extras = ["speedup"], version = "*"}
 gunicorn = "*"
 imap-tools = "*"
 langdetect = "*"
+mysqlclient = "*"
```

Comment:
To be honest I haven't worked with Pip much. After I ran the install and I got the mysql section out, I noticed several other sections were updated as well. In some languages you can just ignore the other changes and move on, but the Pipfile.lock has a `_meta.hash.sha256` section at the top of it, which likely indicates it won't just let me add the mysql piece, for fear of a different hash being generated. How do I address this? I doubt you want a Pipfile.lock change with a bunch of dependent packages being updated.

Reply:
We use `pipenv` instead of regular old `pip`, which manages the Pipfiles and requirements.txt for us (as well as venvs).

I believe the way to add a new requirement is:

1. pipenv install --keep-outdated mysqlclient
2. pipenv lock -r > requirements.txt
That will update the Pipfiles and requirements.txt, with the minimum changes

Reply:
that worked, thanks!